tests/console/validate_first_disk_selection.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate that first disk was selected for installation
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'consoletest';
+
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    select_console 'root-console';
+
+    my @errors;
+    # list info about block devices
+    my @lsblk_output = split(/\n/, script_run('lsblk -n'));
+    # get list of disk names
+    my @disks = map { (split ' ', $_)[0] } grep { /disk/ } @lsblk_output;
+    # get first disk names
+    my $first_disk = shift @disks;
```

Comment:
I'm not a fan of gathering expectations from the system under test, but did you check that disks in the disk selection dialogue are shown always in alphabetical order? I guess we can confirm that with yast team and then it will be fine to rely on lsblk here.

Reply:
For disk the solution is too ugly with test_data, multiple files for simple thing. I was assuming this order. This is the answer from YaST developer:
```

Reply:
I mean, also use same kind of sorting. Test data or just variable would make it easier to understand and change expectations if need to do so.

Reply:
for the answer, unless we would have more than a-z disks, doesn't look like is going to break, lsblk will give us the same order than yast is giving us, am I missing something?

Reply:
Could you please document it. Again, main problem I see that we set expectations from the SUT. For instance, we wrongly select `sdb` in the installer and in SUT there is severe bug, that `sda` is not even mounted. Of course it should be caught by kernel tests earlier, but still.
So if want to keep it like this, please, add the information from Arvin somewhere as a comment, or in the test suite description.

Reply:
I see what you meant now you mentioned the use case, I've added test data to make more robust even it is not very likely that could happen.